### DISCUSSION TITLE
Suggestion: Raven: Switch between notification & widgets view via mouse scroll over notification & widgets button

### DISCUSSION THREAD

**Discussion by sebo505** [BuddiesOfBudgie/budgie-desktop#493] [Ideas]
Currently, to switch in raven from widget view to notification view, you have to click on the notifications button.
In my opinion it would be a handy feature if you could achieve this also via mouse scroll:
 -place mouse cursor anywhere over widgets or notification button
 - scroll up for widgets, scroll down for notifications

**Comment by JoshStrobl** [comment]
Please file this as a feature request in Issues for further discussion.

> you have to click on the notifications button.

By notifications button do you mean the panel widget or switcher? panel widget takes you directly to notifications, so this seems extraneous. If your mouse is already on the switcher, you might as well click rather than scroll.

Also I'd like to note that you can use keyboard shortcuts Super+A and Super+N.
